include/grpcpp/impl/codegen/client_interceptor.h

```diff
@@ -83,10 +82,10 @@ class ClientRpcInfo {
 
   grpc::ClientContext* ctx_ = nullptr;
   const char* method_ = nullptr;
-  grpc::Channel* channel_ = nullptr;
+  grpc::ChannelInterface* channel_ = nullptr;
   std::vector<std::unique_ptr<experimental::Interceptor>> interceptors_;
   bool hijacked_ = false;
-  int hijacked_interceptor_ = false;
+  size_t hijacked_interceptor_ = false;
```

Comment:
This is a bad initialization (mistyped).  I think it is unnecessary, right? If the `hijacked_` variable is false then this index is not important anyway, I think.

Reply:
hahahah. removed!

Reply:
initialization is required though. it complains otherwise